src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -464,6 +502,18 @@ private void UpdateBuckets()
                 if (_transactions.Count > _txPoolConfig.Size)
                     if (_logger.IsWarn) _logger.Warn($"TxPool exceeds the config size {_transactions.Count}/{_txPoolConfig.Size}");
                 _transactions.UpdatePool(_accounts, _updateBucket);
+
+                // ensure the capacity of the blob pool
+                if (_blobTransactions.Count > (_txPoolConfig.PersistentBlobStorageEnabled
```

Comment:
`(_txPoolConfig.PersistentBlobStorageEnabled ? _txPoolConfig.PersistentBlobStorageSize : _txPoolConfig.InMemoryBlobPoolSize)` should be calculated and saved in constructor.

Reply:
Or even better could be moved to _blobPool itself